pkg/patterns/declarative/reconciler.go

```diff
@@ -164,6 +166,26 @@ func (r *Reconciler) reconcileExists(ctx context.Context, name types.NamespacedN
 		return reconcile.Result{}, fmt.Errorf("error applying manifest: %v", err)
 	}
 
+	statusMap := make(map[string]bool)
+	for _, object := range objects.Items {
+		res, err := status.Compute(object.UnstructuredObject())
+		if err != nil {
+			// Do we return errors or just skipp
+		}
+		if res != nil {
+			log.Info("%v %v has status %v, message: %v\n", object.Kind, object.Name, res.Status, res.Message)
+			statusMap[res.Status.String()] = true
```

Comment:
You could make statusMap `map[status.Status]bool` and avoid the `String()` conversion.  That said, I bet they're strings anyway...